ppr-ui/src/components/mhrRegistration/HomeOwners/AddEditHomeOwner.vue

```diff
@@ -1,5 +1,5 @@
 <template>
-  <v-card flat class="py-6 px-8 rounded">
+  <v-card flat class="py-6 px-8 mb-20 rounded">
```

Comment:
`mb-20` is out of range of the helper classes. I think the max helper class is 10.

Reply:
To add, each increment here is worth about 4px by Vuetify definition. 
ie `mb-1` === 4px margin bottom and so on

Reply:
Thanks for letting me know, was unaware!